### PR TITLE
Added travis-ci and PyPI badges #6827

### PR DESCRIPTION
**Pull request by rajatguptarg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 trytond-attachment-s3
 =====================
 
-Amazon S3 backend for Tryton Attachments
\ No newline at end of file
+Amazon S3 backend for Tryton Attachments
```
Comment (PritishC) [inline README.md]: This line shouldn't have been affected by your patch, you added your lines under this.

Could you please fix this? Use `git diff`.

**Comment by PritishC** [comment]
Add task ID to commit message.

Also, the diff is still not right.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+
+
```
Comment (PritishC) [inline README.md]: Remove this extra line, otherwise it LGTM.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
 trytond-attachment-s3
 =====================
-
-Amazon S3 backend for Tryton Attachments
\ No newline at end of file
+Amazon S3 backend for Tryton Attachments
+[![Build Status](https://travis-ci.org/openlabs/trytond-attachment-s3.svg)](https://travis-ci.org/openlabs/trytond-attachment-s3)
```
Comment (PritishC) [inline README.md]: Blank line required here.

Comment (PritishC) [inline README.md]: One blank line here, or else the Amazon S3 line and the badges appear in the same line.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,8 @@
 trytond-attachment-s3
 =====================
 
-Amazon S3 backend for Tryton Attachments
\ No newline at end of file
+Amazon S3 backend for Tryton Attachments
+[![Build Status](https://travis-ci.org/openlabs/trytond-attachment-s3.svg)](https://travis-ci.org/openlabs/trytond-attachment-s3)
```
Comment (PritishC) [inline README.md]: One blank line here, or else the Amazon S3 line and the badges will appear in the same line.

**Comment by PritishC** [comment]
LGTM.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,9 @@
 trytond-attachment-s3
 =====================
 
-Amazon S3 backend for Tryton Attachments
\ No newline at end of file
+Amazon S3 backend for Tryton Attachments
+
+[![Build Status](https://travis-ci.org/openlabs/trytond-attachment-s3.svg)](https://travis-ci.org/openlabs/trytond-attachment-s3)
```
Comment (aroraumang) [inline README.md]: For develop branch

**Comment by aroraumang** [comment]
LGTM

**Pull request merged**